test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -631,6 +633,14 @@ class TestDapp {
     });
   }
 
+  async check_ethSignErrorMessage() {
+    const ethSignButton = await this.driver.findElement(this.ethSignButton);
+    const expectedMessage =
+      'ERROR: THE METHOD "ETH_SIGN" DOES NOT EXIST / IS NOT AVAILABLE.';
+    const actualText = await ethSignButton.getText();
+    assert.equal(actualText, expectedMessage);
```

Comment:
this is an anti-pattern that we should avoid.
Instead of finding a selector and then trying to assert the expected value, we should try to find the element by its expected value.
Otherwise this can lead to race conditions, where the element is already in the DOM but with a different value (ie default) for some brief ms, making the assertion fail

Reply:
Additionally, we could move the selector above, in the selectors section. Something like:

```
ethSignErrorMessage = {
  testId: 'ethSign'
  text: 'ERROR: THE METHOD "ETH_SIGN" DOES NOT EXIST / IS NOT AVAILABLE.'`
}

```

and then use this selector in the method